packages/common/src/discussions/api/types.ts

```diff
@@ -844,38 +824,54 @@ export interface ICreateChannel
  * @extends {IWithTimestamps}
  */
 export interface IChannel extends IWithAuthor, IWithEditor, IWithTimestamps {
-  access: SharingAccess;
+  id: string;
+  access: SharingAccess | null;
+  allowAnonymous: boolean | null;
   allowAsAnonymous: boolean;
-  allowAnonymous: boolean;
   allowedReactions: PostReaction[] | null;
   allowReaction: boolean;
   allowReply: boolean;
   blockWords: string[] | null;
   channelAcl?: IChannelAclPermission[];
   defaultPostStatus: PostStatus;
-  groups: string[];
+  groups: string[] | null;
   metadata: IChannelMetadata | null;
-  id: string;
   name: string | null;
   orgId: string;
-  orgs: string[];
+  orgs: string[] | null;
   posts?: IPost[];
   softDelete: boolean;
 }
 
 /**
- * parameters/options for updating channel settings
+ * parameters for updating a channel
  *
  * @export
  * @interface IUpdateChannel
- * @extends {ICreateChannelSettings}
+ * @extends {IUpdateChannelSettings}
  * @extends {IUpdateChannelPermissions}
- * @extends {Partial<IWithAuthor>}
  */
 export interface IUpdateChannel
-  extends ICreateChannelSettings,
-    IUpdateChannelPermissions,
-    Partial<IWithAuthor> {}
+  extends IUpdateChannelSettings,
+    IUpdateChannelPermissions {}
```

Comment:
This relates to the legacy V1 code creating channels on the fly in private content discussions. The desire was that the channel owner should be the same as the group owner, so the client would override/update the channel owner to mirror that of the group that was configured for the channel. We'll need to kill off that owner reassignment code when we strip out the ability to create channels on the fly.